.travis.yml

```diff
@@ -0,0 +1,17 @@
+sudo: required
```

Comment:
Question:  Do .yml files require licenses and headers?  I would think the answer is YES. ??

Reply:
i added apache license to this file too as this yaml file contains shell script snippets.